packages/components/src/internal/components/samples/DiscardConsumedSamplesPanel.tsx

```diff
@@ -28,7 +28,7 @@ export const DiscardConsumedSamplesPanel: FC<Props> = memo(props => {
                                 onChange={toggleShouldDiscard}
                                 checked={shouldDiscard}
                             />
-                            <span className="left-spacing">{discardTitle}</span>
+                            <span className="discard-consumed-title left-spacing">{discardTitle}</span>
```

Comment:
I have a feeling we will be doing a lot of this, i.e. adding a className, for the enzyme to RTL conversion.